nixos/tests/magic-wormhole-mailbox-server.nix

```diff
@@ -0,0 +1,18 @@
+import ./make-test-python.nix ({ pkgs, ... }: {
+  name = "magic-wormhole-mailbox-server";
+  meta = with pkgs.stdenv.lib.maintainers; {
+    maintainers = [ mmahut ];
+  };
+
+  nodes = {
+    machine = { ... }: {
+      services.magic-wormhole-mailbox-server.enable = true;
+    };
+  };
+
+  testScript = ''
+    start_all()
+    machine.wait_for_unit("magic-wormhole-mailbox-server.service")
+    machine.wait_for_open_port(4000)
```

Comment:
i would find it pretty cool if this test also tested if the service works with clients. That might make this integration test become a pretty cool demo for how nixos integration tests work, actually.

Reply:
@tfc great suggestion.